gnovm/cmd/gno/scripts_test.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"strconv"
+	"testing"
+
+	"github.com/gnolang/gno/gnovm/pkg/integration"
+	"github.com/rogpeppe/go-internal/testscript"
+	"github.com/stretchr/testify/require"
+)
+
+func TestScript(t *testing.T) {
+	updateScripts, _ := strconv.ParseBool(os.Getenv("UPDATE_SCRIPTS"))
```

Comment:
we already have a flag for this, `-update-golden-files` or `-update-golden-tests` iirc. it should be exposed as a global unexported var.

Reply:
can we do this in another PR? this was carried over from the existing files (test_test.go, transpile_test.go) and I think there are probably more tests to update to use that flag